README.md

```diff
@@ -15,7 +15,7 @@
 </p>
 
 > ⚠️ **NOTE** ⚠️  
-> `PactSwift` is under heavy development.
+> `PactSwift` is under heavy development. We are specifically looking into better support for `aarch64` along with `x86_64`.
```

Comment:
Small typo
```suggestion
> `PactSwift` is under heavy development. We are specifically looking into better support for `arch64` along with `x86_64`.
```

Reply:
Hrm… no I think `aarch64` is correct. [See here](https://developer.arm.com/architectures/learn-the-architecture/aarch64-instruction-set-architecture/registers-in-aarch64-general-purpose-registers)

Reply:
Yes, you are right it is `aarch64`, but wouldn't be better to use `arm64` as that is the value that you configure in Xcode Build Settings  ?

Reply:
Yes maybe that would be best :)

Reply:
I was referring to Rust triple that is `aarch64-apple-darwin`. But since I didn't mention that that is the reason for this note, `arm64` should be used as that is the code for Apple Silicon architecture. 👍